SophiaWorkflow.iml

```diff
@@ -18,10 +18,11 @@
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
     <orderEntry type="library" name="Gradle: org.codehaus.groovy:groovy-all:2.4.9" level="project" />
-    <orderEntry type="module" module-name="Roddy" />
-    <orderEntry type="module" module-name="RoddyToolLib" />
-    <orderEntry type="module" module-name="COWorkflowsBasePlugin_1.4.0" />
-    <orderEntry type="module" module-name="PluginBase" />
     <orderEntry type="library" name="Python 3.7 (base) interpreter library" level="application" />
+    <orderEntry type="module" module-name="COWorkflowsBasePlugin_1.4.1" />
+    <orderEntry type="module" module-name="Roddy.main" />
```

Comment:
i was not aware ot the *.main convention. what are the implications/prerequisites of using *.main?

Reply:
Since recently (cannot give the exact version number) IntelliJ refuses to keep the old settings for Gradle-based projects, and instead reinstalls modules using the *.main and *.test convention. I actually had to set up my complete development environment because of this issue, and it seemed senseless to me, to "swim against the stream" and instead accepted they way IntelliJ does this now.

Reply:
IC - thanks for clarifying